src/cpp/include/openvino/genai/generation_config.hpp

```diff
@@ -142,6 +142,8 @@ static constexpr ov::Property<float> presence_penalty{"presence_penalty"};
 static constexpr ov::Property<float> frequency_penalty{"frequency_penalty"};
 static constexpr ov::Property<size_t> rng_seed{"rng_seed"};
 
+static constexpr ov::Property<SchedulerConfig> enable_continuous_batching{"enable_continuous_batching"};
```

Comment:
why is  part of generation_config.hpp ? I suppose llm_pipeline is the right place.